tests/driver_blemin/Makefile

```diff
@@ -0,0 +1,18 @@
+APPLICATION = driver_blemin
```

Comment:
we should not mix up "blemin", "ble_ll" and "nrf51_ble". I'm currently not sure but "blemin" is the old stuff that could be removed (in general), right? Correct me if I'm wrong!